### ISSUE TITLE
Setup test group for experimenting on FF

### ISSUE THREAD

**Issue by bmann** [Finhaven/blockchainYVR#3]
We can setup a test group / forum on FF to see if it works.

**Comment by bmann** [comment]
OK, I have set it up here https://community.frontierfoundry.co/c/blockchainyvr -- that's the category. We can create sub-categories if needed, but probably best to start with one category.

I have set permissions so everyone (including not logged in) can see the contents of the category.

Only members of the blockchainYVR group can post or reply.

@chiselinc you already had an account, so I made you an owner of the group https://community.frontierfoundry.co/groups/blockchainyvr

@please-wait you get to do the full sign up test! Create an account, and then you should be able to go to the groups link and add yourself to the group. Once you've added yourself to the group, then you can post to the category. Maybe a little confusing? We'll see!

The group also functions as a sort of mailing list. Only admins, owners, moderators can message -- which will be default notify everyone in the group.

There is an Android app https://play.google.com/store/apps/details?id=com.discourse&hl=en and an iOS app https://itunes.apple.com/ca/app/discourse-app/id1173672076?mt=8

You can also set notification settings per topic/tag/etc including muting.

**Comment by please-wait** [comment]
Making the account was a breeze, I just don't have permission to post/reply under BlockchainYVR category.

I can post under some other categories though. Is it because of the trust level?

**Comment by bmann** [comment]
@please-wait you need to add yourself to the group first! https://community.frontierfoundry.co/groups/blockchainyvr

As soon as you add yourself, then you can post.

Let me know if that works!

**Comment by please-wait** [comment]
Ah @bmann now I see what you meant by groups... Yeah, the UI is not intuitive for finding the groups section. Does Discourse have any customization to help with this? Like adding a "join this group" button?

If not, maybe we can put a sticky post above everything else, "Join BlockchainYVR group" with this link?

**Comment by please-wait** [comment]
Yup, they say put it in the description: https://meta.discourse.org/t/groups-link-to-group/58880

**Comment by bmann** [comment]
I edited the description so that it shows.

See https://community.frontierfoundry.co/categories for a list of all categories which shows the group in the description. Also: I'll re-order the categories to BlockchainYVR shows up higher on the list.

I bolded it in the Welcome post too which will remain pinned https://community.frontierfoundry.co/c/blockchainyvr

I am going to consider this closed for now.
